### ISSUE TITLE
Error on smtp-faker test

### ISSUE THREAD

**Issue by vadim-su** [zzzsochi/smtp-faker#1]
```
ERROR:asyncio:Task exception was never retrieved
future: <Task finished coro=<MessageHandler.msg_received() done, defined at /home/vagrant/src/smtp_faker/smtpd.py:33> exception=RuntimeError('Task got bad yield: <aiohttp_traversal.traversal.Traverser object at 0x7f2d81a3c7f0>',)>
Traceback (most recent call last):
  File "/usr/lib/python3.5/asyncio/tasks.py", line 241, in _step
    result = coro.throw(exc)
  File "/home/vagrant/src/smtp_faker/smtpd.py", line 35, in msg_received
    root = yield from self.app.get_root()
RuntimeError: Task got bad yield: <aiohttp_traversal.traversal.Traverser object at 0x7f2d81a3c7f0>
```

**Comment by zzzsochi** [comment]
Fixed in c7e21e7e3b1f3f4a20e4d9926631d9d3645d317d.

**Comment by charisz** [comment]
In this case perhaps you should modify the setup.py file

       classifiers=[
        "License :: OSI Approved :: BSD License",
        "Operating System :: POSIX",
        "Programming Language :: Python :: 3.5",
        "Topic :: Internet :: WWW/HTTP",
    ],
